README.md

```diff
@@ -0,0 +1,85 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+iOS Playbook 📚
+==================================
+
+At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview on how we work:
```

Comment:
```suggestion
At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview on how we work:
```
```suggestion
```
At Babylon, **transparency** is a core value that we believe should be present in everything we do. This playbook embodies it by giving an overview of how we work:
```
```